services/QuillLessonsServer/bootstrap.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+echo "Installing script dependencies..."
+brew install jq
+
+echo "Installing RethinkDB for local use..."
+brew update
+brew install rethinkdb
+
+echo "Setting up local RethinkDB in prep to load seed data..."
+./rethink_local.sh start
+echo -n "Please provide the path to your RethinkDB seed data dump: "
```

Comment:
Reading the [rethink docs](https://www.rethinkdb.com/docs/importing/), this import will assume the format is .json (`--format: CSV or JSON (default JSON)`), so we should include directions about the download needing to be .json.

Reply:
There are no options in the downloads available from Compose.io.  It just provides a `.tar` file which is, presumably in JSON format.

Reply:
Yup, looking at the tarball decompressed, it's folders of schema and data in `.json` format. Let's just add a comment somewhere that this file is expected to be json (in case they change the format to `csv` in the future, at which time, this would bomb out).

Reply:
Seems reasonable.  I'll add a note in the section of the README that discusses downloading seed data.